docs/sources/mimir/configure/configure-spread-minimizing-tokens/_index.md

```diff
@@ -0,0 +1,108 @@
+---
+aliases:
+  - ../operators-guide/configure/configure-spread-minimizing-tokens/
+description: Learn how to migrate ingesters to spread-minimizing tokens.
+weight: 115
+menuTitle: Spread-minimizing tokens
+title: Migrate ingesters to spread-minimizing tokens
+---
+
+# Migrate ingesters to spread-minimizing tokens
+
+Using this guide, you can configure Mimir's ingesters to use the _spread-minimizing token generation strategy_ .
```

Comment:
You have a good open. I tried to remove the passive voice. What do you think? Also, maybe include a sentence about why someone would want to use this and when. 

The suggestion below just removes an extra space before the period. 
```suggestion
Using this guide, you can configure Mimir's ingesters to use the _spread-minimizing token generation strategy_.
```